Dockerfile

```diff
@@ -29,8 +29,7 @@ RUN apt-get update && \
         && \
     rm -rf /var/lib/apt/lists/* 
 
-ARG driver-version=390.116
-ENV NVIDIA_DRIVER_VERSION=$driver-version
+ENV NVIDIA_DRIVER_VERSION="440.33"
```

Comment:
Set version statically in the ENV VAR. I created a script to replace all appearances of this version var